src/pages/invoices/common/hooks/useFormatMoney.ts

```diff
@@ -91,6 +91,6 @@ export function useFormatMoney(props: Props) {
       return Number.formatMoney(value, currency, country);
     }
 
-    return value;
+    return parseFloat(value.toString()).toFixed(2);
```

Comment:
@turbo124 is this okay? We are hardcoding the decimal rounding to 2nd **all the time**?

Reply:
@turbo124 @beganovich Just to mention from my end. This is the case when we don't have currency and country, so we don't have separators and similar things to round the value. I'm not sure we have any additional information here to round it dynamically. So that's the reason for hard coding, otherwise without a round UI it really doesn't look good.

Reply:
@beganovich @Civolilah no, we should never hardcode the rounding, it shuld always use the formatMoney() method.

Reply:
@turbo124 @beganovich We talked on Slack about this scenario and I couldn't replicate it. So I just changed the code to the old version where we don't have this manual fix. If we catch the scenario through testing in the future, it will be a minor job to fix it.